renkuaqs/plugin.py

```diff
@@ -249,21 +259,25 @@ def _param_value(rdf_iteral):
         ?run <http://odahub.io/ontology#isRequestingAstroObject> ?a_object;
              <http://odahub.io/ontology#isUsing> ?aq_module;
              ^oa:hasBody/oa:hasTarget ?runId .
+             
         ?a_object <http://purl.org/dc/terms/title> ?a_object_name .
+        
         ?aq_module <http://purl.org/dc/terms/title> ?aq_module_name .
+        
         ?run ?p ?o .
-
+        
+        FILTER (!CONTAINS(str(?a_object), " ")) .
```

Comment:
I do not think they should be silently ignored. In fact, they should not be created. If created and available in the graph, it should be detected and warning shown with. While also showing the entires themselves in the table.

Silently ignoring means that something that's erronenously created will be mysteriously missing during inspection, but will continue to contaiminate the store.

Reply:
A warning is actually already shown, and the table can actually be created. 

It comes directly from renku when querying the graph:

```
WARNING:rdflib.term:file://~/.renku/Mrk 421 does not look like a valid URI, trying to serialize this will break.
WARNING:rdflib.term:file://~/.renku/Mrk 421 does not look like a valid URI, trying to serialize this will break.
```

Reply:
However, these create issues when serializing afterwards, that is why they cannot be included.

Reply:
ok, maybe let's add another warning clearly stating that the table is incomplete, the store is corrupted, advice what to do: recreate it or so.